features/support/env.rb

```diff
@@ -17,33 +17,22 @@
 Capybara.app_host = ENV["GOVUK_WEBSITE_ROOT"]
 phantomjs_logger = File.open("log/phantomjs.log", "a")
 
-BLACKLISTED_URLS = ['www.google-analytics.com']
+GOOGLE_ANALYTICS_URL = 'www.google-analytics.com'
+BLACKLISTED_URLS = [GOOGLE_ANALYTICS_URL]
 
 Capybara.register_driver :poltergeist do |app|
-  Capybara::Poltergeist::Driver.new(app, phantomjs_logger: phantomjs_logger)
+  options = {
+    debug: ENV['POLTERGEIST_DEBUG'] || false,
+    phantomjs_logger: phantomjs_logger,
+    url_blacklist: BLACKLISTED_URLS
```

Comment:
I'm now completely confident that I haven't reintroduced the problem described in issue #256. I've updated the commit titled "Instantiate Poltergeist driver with url_blacklist" to explain the change in more detail.